pipelines/build/regeneration/jdk14_regeneration_pipeline.groovy

```diff
@@ -1,5 +1,8 @@
 import java.nio.file.NoSuchFileException
```

Comment:
Lots of repeated code in jdk<version>_regeneration_pipeline files - maybe fix after this PR?

Reply:
Existing issue to do just that -> https://github.com/AdoptOpenJDK/openjdk-build/issues/2290